modules/module.html

```diff
@@ -0,0 +1,45 @@
+<!--
+module.html - demonstrate using ilib modules in webpack
+
+Copyright © 2022 JEDLSoft
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html>
+  <head>
+    <title>Ilib Module Test</title>
+    <script>
+    // don't need logging output
+    var log4js = null;
+    var NodeLoader = null;
+    </script>
+    <script src="./test/moduleTest.js"></script>
+  </head>
+  <body>
+    <h1>iLib Module Test</h1>
+    <script>
+    var LocaleInfo = LocaleInfoModule.default;
```

Comment:
where did this `LocaleInfoModule` name come from?

Reply:
From the webpacked test file.  See `webpack.config.js` line 30.